### ISSUE TITLE
Flex games should have betting time restrictions updated correctly

### ISSUE THREAD

**Issue by LaCuneta** [SASEliminator/Eliminator#8]
In prior seasons (I didn't get a chance to check in this one), when a game is flexed to the late start time on Sunday, the dates in the system don't change.

- Game 1, previously 12pm, now 7pm, cannot be picked/bet-on properly after 12pm.
- Game 2, previously 7pm, now 3:30pm, can be picked/bet-on after it has been played, allowing for guaranteed victories/wins.

If this is, indeed, still an issue, then the game times should be updated when flexed so the bets/picks can be done correctly.

**Comment by qwijib0** [comment]
Yes, we intend to have flexed game times correctly adjusted! this is a thing that we had talked about as a need-to-fix.

**Comment by multigl** [comment]
The old Ruby site did not handle flex games. It compared your bet against whatever kickoff time it had on file, and that file was only as fresh as the last time it was updated. A Sunday night game moved up to 3:30 stayed open for betting until 7pm, so anyone who noticed could bet a game already in its third quarter and take a sure thing. A noon game pushed to prime time locked at noon, which just meant you lost six hours you should have had. 

The Django version stores no deadline at all. Every pick, straight bet and parlay leg is checked against the game's current kickoff time at the moment you hit submit, plus the same five minute grace period the old site had, so when we re-running the schedule reader it is completely fixed. Once the new time is in, both cases are simply correct: the moved-up game locks at 3:30, the moved-back game stays open until 7.
